lib/fastlane/actions/mailgun.rb

```diff
@@ -26,13 +22,10 @@ def self.description
 
       def self.available_options
         [
-          FastlaneCore::ConfigItem.new(key: :mailgun_sandbox_domain,
```

Comment:
Sandbox domain could be inferred from postmaster .. The other fields names
ive refactored cause i've think i didnt name than properly , its a bit
redundant. But i could keep than as before if you prefer . I understand
that this will broke the older builds , but people can check the
documentation and update there setup . Do you think is too much work for
such small change ?

On Thu, Jun 18, 2015 at 1:23 AM Felix Krause <notifications@github.com>
wrote:

> In lib/fastlane/actions/mailgun.rb
> <https://github.com/KrauseFx/fastlane/pull/355#discussion_r32698132>:
>
> > @@ -26,13 +22,10 @@ def self.description
> >
> >        def self.available_options
> >          [
> > -          FastlaneCore::ConfigItem.new(key: :mailgun_sandbox_domain,
>
> This will invalidate all existing setups using this integration, right? Is
> there a reason you renamed mailgun_sandbox_postmaster to postmaster?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/KrauseFx/fastlane/pull/355/files#r32698132>.
>